lib/destination/ddl/ddl_test.go

```diff
@@ -1,123 +1,40 @@
-package ddl_test
+package ddl
```

Comment:
Note the pkg name change. I moved all the other tests out so that we can have this be `ddl`. That way, we can have `shouldCreatePrimaryKey` remain a private function.

Once this PR merges, I'm going to try to remove the need for us to specify a different package name.